### PR TITLE
Evelinabe - Happy thoughts

### PR DESCRIPTION
**Pull request by evelinabe**
Link to my deployed project: https://happiest-happy-thoughts.netlify.app/

My reflections on how this week's project turned out:
I'm still lagging behind. The React part of the blue level is done, I will continue on with the styling.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+  const [thoughts, setThoughts] = useState([]);
+
+  const fetchThoughts = () => {
+    fetch(THOUGHTS_URL)
+      .then((res) => res.json())
+      .then((thought) => setThoughts(thought))
+      .catch((error) => console.error(error));
+  }
+
+  const postThoughts = (newThought) => {
```
Comment (mats-johansen) [inline code/src/App.js:17]: You don't need () when passing only one argument here. On the other hand the code almost look's better with (newThought) then newThought :D

File: code/src/components/Thought.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        {message}
+        <span className="thought-time">{time}</span>
+        <span className="liked-container">
+          <ThoughtLikes id={id} onLikedThought={onLikedThought} />
+          <span className="liked"> x {hearts}</span>
+        </span>
+      </p>
+    </div>
+  );
+};
```
Comment (mats-johansen) [inline code/src/components/Thought.js:23]: Very clean code in all the components!

File: code/src/components/ThoughtLikes.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const ThoughtLikes = ({ id, onLikedThought }) => {
+  return (
+    <button type="button" onClick={() => onLikedThought(id)}>
+      <span role="img" aria-label="red heart">❤️</span>
+    </button>
+  )
```
Comment (mats-johansen) [inline code/src/components/ThoughtLikes.js:8]: ;

**Review by mats-johansen — APPROVED** [review]
Great job! It look's look's and works great. I see no problems with the code or the functionality besides when writing a long message without any whitespace, like this:
ddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddddda

It displays this outside the div. It's best to try it for yourself because of my bad explanation . I think adding word-wrap:break-word; in your css will do the job. 

Once again, great job!

File: code/src/components/Thought.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+export const Thought = ({
+  message,
+  hearts,
+  time,
+  id,
+  onLikedThought
+}) => {
+  return (
+    <div className="thought-container">
+      <p key={id} id={id}>
```
Comment (EmOlsz) [inline code/src/components/Thought.js:13]: You don't need to use `key` prop here, we are not performing any loop

File: code/src/App.js (hunk 1 above)
Comment (evelinabe) [reply code/src/App.js:17]: Tried to remove it but then it wouldn't compile :(

File: code/src/components/Thought.js (hunk 4 above)
Comment (evelinabe) [reply code/src/components/Thought.js]: Removed.

**Pull request closed without merging**